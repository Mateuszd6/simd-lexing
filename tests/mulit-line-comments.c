/**
 * This is a doxygen-like comment with not valuable information at all, but at
 * least it's long so our code looks "better" I guess and the auto-generated
 * shit-worth documentation that we give to our clients in absurd amounts will
 * look more professional, because it will have more text. Who cares that nobody
 * will ever read that.
 */ hello cruel world!
