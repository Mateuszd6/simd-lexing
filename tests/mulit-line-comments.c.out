../tests/mulit-line-comments.c:7:5: TOK 0 (L = 5): "hello"
../tests/mulit-line-comments.c:7:11: TOK 0 (L = 5): "cruel"
../tests/mulit-line-comments.c:7:17: TOK 0 (L = 5): "world"
../tests/mulit-line-comments.c:7:22: TOK 1 (L = 1): "!"
Parsed: 8 lines, 3 ids, 0 strings, 0 chars, 0 ints, 0 hex,   0 floats,    0 //s, 1 /**/s,   0 #foo
